components/11textut.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/docs.css">
+    <title>Even UI | Texts</title>
+    <script src="https://kit.fontawesome.com/9f366bbf8a.js" crossorigin="anonymous"></script>
+    <script src="./index.js"></script>
+</head>
+
+<body>
+    <nav class="topnav" id="myTopnav">
+        <p class="nav-itm nav-itm-text active "><a href="../index.html">Home</a> </p>
+        <p class="nav-itm right "> <span>Github</span> <i class="fab fa-github"></i></p>
+        <p class="nav-itm right "> <span>Repository</span> <i class="fas fa-code-branch"></i></i></p>
+        <p class="nav-itm control right" onclick="myFunction()" > <i class="fa fa-bars"></i></p>
+    </nav>
```

Comment:
it'd be great if you remove this repeating code and make it work like 

```
const topnav = document.getElementByClassName('topnav')[0]
document.addEventListener('DOMContentLoaded', () => {
    content = "<p class="nav-itm nav-itm-text"><a href="../index.html">Home</a><p>
   // add the remaining elems here
   ";

   topnav.innerHTML = content
})
```

benefits : easy to change at all place, follows DRY